src/uikit.cs

```diff
@@ -277,21 +281,27 @@ interface NSDataAsset : NSCopying
 		NSString TypeIdentifier { get; }
 	}
 
+	delegate void NSFileProviderExtensionFetchThumbnailsHandler (NSString identifier, [NullAllowed] NSData imageData, [NullAllowed] NSError error);
+
 	[NoWatch]
 	[NoTV]
 	[iOS (8,0)]
 	[ThreadSafe]
 	[BaseType (typeof (NSObject))]
 	partial interface NSFileProviderExtension {
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'NSFileProviderManager' instead.")]
 	    [Static, Export ("writePlaceholderAtURL:withMetadata:error:")]
 	    bool WritePlaceholder (NSUrl placeholderUrl, NSDictionary metadata, ref NSError error);
 	
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'FileProvider.NSFileProviderManager.GetPlaceholderUrl (NSUrl)' instead.")]
```

Comment:
really `FileProvider.NSFileProviderManager.GetPlaceholderUrl `  ?
that seems to be a static method so the call should be `NSFileProviderManager.GetPlaceholderUrl `